.github/workflows/test-yjit.yaml

```diff
@@ -32,3 +32,33 @@ jobs:
           bundler: latest # needed to fix issue with steep on Ruby 3.0/3.1
           cache-version: v2 # bump this to invalidate cache
       - run: bundle exec rake spec:yjit
+  test-yjit-stats:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        ruby:
+          - '3.3'
+          # ADD NEW RUBIES HERE
+    name: Test YJIT Stats (${{ matrix.os }}, ${{ matrix.ruby }})
+    runs-on: ${{ matrix.os }}
+    env:
+      RUBYOPT: "--yjit --yjit-stats=quiet"
```

Comment:
I just copied the job, seemed easier/more maintainable than trying to fit this logic into the matrix using a single job.

Also, I don't see these actions in https://github.com/DataDog/dd-trace-rb/actions, so I'm guessing they need to be approved by a maintainer first.

Reply:
Yeah, we need to allow it on our side! I'll press the magic button :)

That said, I've suggested a matrix solution for this -- while I don't think it's the best solution every time, it does seem relatively OK on this case (vs having to maintain this whole other config?)